src/Hl7.Fhir.Support.Poco/Rest/HttpToEntryExtensions.cs

```diff
@@ -30,7 +30,7 @@ internal static EntryResponse ToEntryResponse(this HttpResponseMessage response,
                 Status = ((int)response.StatusCode).ToString(),
                 ResponseUri = response.RequestMessage.RequestUri,//this is actually the requestUri, can't find the responseUri
                 Body = body,
-                Location = response.Headers.Location?.AbsoluteUri ?? response.Content.Headers.ContentLocation?.AbsoluteUri,
+                Location = response.Headers.Location?.ToString() ?? response.Content.Headers.ContentLocation?.ToString(),
```

Comment:
Better to use OriginalString. See also some background at https://docs.microsoft.com/en-us/dotnet/api/system.uri.originalstring?view=net-5.0#remarks